src/main/webapp/resources/js/components/tabs/SideTabs.jsx

```diff
@@ -0,0 +1,40 @@
+/**
+ * Component to render styled side tabs. Use
+ * this whenever vertical tabs within a page
+ * are required
+ */
+
+import React from "react";
+import PropTypes from "prop-types";
+import { Tabs } from "antd";
+import styled from "styled-components";
+
+const VerticalTabs = styled(Tabs)`
+  .ant-tabs-tab {
+    @media only screen and (min-width: 800px) {
+      width: 200px;
+    }
+  }
+`;
+
+export function SideTabs(props) {
+  return (
+    <VerticalTabs
+      type={props.type || "card"}
+      activeKey={props.activeKey}
+      onChange={props.onChange}
+      tabPosition={props.tabPosition || "left"}
+      animated={props.animated || false}
+    >
+      {props.children}
+    </VerticalTabs>
+  );
+}
+
+SideTabs.propTypes = {
+  type: PropTypes.string,
+  activeKey: PropTypes.string,
+  onChange: PropTypes.func,
+  tabPosition: PropTypes.string,
+  animated: PropTypes.bool
```

Comment:
Don't forget `children` inhere.  You also should add to your docs what the props are.  Try following https://devdocs.io/jsdoc/

Reply:
Fixed in 5e42840cf8b9aa72a4ab4384c86a925e103d0382 and 96a30eef5b4770f8733922e46814ef78f89cc36e